inc/saiswitch.h

```diff
@@ -1444,8 +1444,10 @@ typedef enum _sai_switch_attr_t
      *
      * Hardware information format is based on SAI implementations by vendors.
      * String is NULL terminated. Format is vendor specific.
-     * Example: Like PCI location, I2C address etc.
+     * Example: Like PCI location, I2C address, MDIO bus SysFS information etc.
      * In case of NULL, First NPU attached to CPU will be initialized.
+     * For the MDIO SysFS driver support, the interface name and phy_id should be
```

Comment:
For non sysfs MDIO, I think this statement should also be added as it is not always we are using mdio sysfs.
For non Sysfs MDIO, only phy_id should be provided.